test/js/spec/test-main.js

```diff
@@ -1,18 +1,45 @@
-var tests = ['modules/checkout/regex.spec'];
+var tests = [];
+
+for (var file in window.__karma__.files) {
+    if (window.__karma__.files.hasOwnProperty(file)) {
+        if (/\.spec\.js$/.test(file)) {
+            tests.push(file);
+        }
+    }
+}
+
 
 // docs: http://karma-runner.github.io/0.8/plus/RequireJS.html
 requirejs.config({
-    // Karma serves files from '/base'
-    baseUrl: '/base/test/js/spec',
+    /**
+     * Karma serves files from '/base'
+     */
+    baseUrl: '/base/assets/javascripts/',
 
-    // Keep these in sync with the paths found in the requireJs paths
+    /**
```

Comment:
Yeah, had the same thought on Membership. Not a great way that I've  found so far, unless I can find a way to share a json file between the two.